zero_sum_eval/games/chess/chess_game.py

```diff
@@ -66,24 +61,23 @@ def validate_game(self):
             return "Fivefold repetition"
         elif not self.board.is_valid():
             return "Invalid"
-        else:
-            return self.context['message']
+        return None
```

Comment:
Are you referring to this @c-rit ?
The reason I formulated it like this is because (i) previously None could never be returned (which contradicts the specs) and (ii) the caller already has access to `self.context['message']` and doesn't get any new info from this return.

The behavior now is, if some end game condition is met then we return something, otherwise we return None and let the game continue.

Also, I am considering renaming this method as well. `validate` doesn't capture what this does.